Palindrome.java

```diff
@@ -0,0 +1,17 @@
+class PalindromeExample{  
+ public static void main(String args[]){  
+  int r,sum=0,temp;    
+  int n=454;//It is the number variable to be checked for palindrome  
+  
+  temp=n;    
+  while(n>0){    
```

Comment:
There are no Tests for this code, which makes it hard to maintain later and to verify now